draft-ietf-tls-esni.md

```diff
@@ -600,6 +644,8 @@ error code.
 
 ### HelloRetryRequest
 
+[[OPEN ISSUE: How does "ech_confirm" impact HRR logic?]]
```

Comment:
HRR right now is quite a headache if the client doesn't match the inner and outer key_share and cipher suite preferences (to the point that I'd argue we should mandate that in the current form). If we're going the new extension route, I think it makes sense to put the indicator in HRR too. But we can sort that out separately.